spec/features/user_views_past_orders_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe "User can view past orders" do
+  let(:user) { User.create(username: "adam", password: "adam") }
+
+  it "user sees a profile page" do
+    visit '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.